### DISCUSSION TITLE
How do I solve Thread 1: EXC_BREAKPOINT (code=1, subcode=0x1800f4978) error?

### DISCUSSION THREAD

**Discussion by JansonJS** [sst-inc/help#13] [Q&A - iOS] [answered]
**So basically I'm doing my code and this error: "Thread 1: EXC_BREAKPOINT (code=1, subcode=0x1800f4978)" pops up in the "AppDelegate.swift" tab. Here is the code I have entered in ViewController.swift:**

import UIKit

class ViewController: UIViewController {
    
    var counter = 0
    @IBOutlet weak var countlabel: UILabel!
    override func viewDidLoad() {
        super.viewDidLoad()
        // Do any additional setup after loading the view.
        countlabel.text = "\(counter)"
        
    }
    
    @IBAction func buttoncounter(_ sender: Any) {
        counter += 1
        countlabel.text = "\(counter)"
    }
    

}

**and here's the code in AppDelegate.swift (the error occured at @main)**
 
import UIKit

@main
class AppDelegate: UIResponder, UIApplicationDelegate {

    func application(_ application: UIApplication, didFinishLaunchingWithOptions launchOptions: [UIApplication.LaunchOptionsKey: Any]?) -> Bool {
        // Override point for customization after application launch.
        return true
    }

    // MARK: UISceneSession Lifecycle

    func application(_ application: UIApplication, configurationForConnecting connectingSceneSession: UISceneSession, options: UIScene.ConnectionOptions) -> UISceneConfiguration {
        // Called when a new scene session is being created.
        // Use this method to select a configuration to create the new scene with.
        return UISceneConfiguration(name: "Default Configuration", sessionRole: connectingSceneSession.role)
    }

    func application(_ application: UIApplication, didDiscardSceneSessions sceneSessions: Set<UISceneSession>) {
        // Called when the user discards a scene session.
        // If any sessions were discarded while the application was not running, this will be called shortly after application:didFinishLaunchingWithOptions.
        // Use this method to release any resources that were specific to the discarded scenes, as they will not return.
    }

}

**I don't know how to fix this problem so please help.**
<img width="1440" alt="Screenshot 2021-07-29 at 9 03 21 PM" src="https://user-images.githubusercontent.com/88146504/127496908-16cddc98-5328-4348-a98a-6e8b2246bb02.png">

**Comment by Ethan-Chew** [comment]
Hi Janson, can you attach a screenshot of your ViewController.swift file? Thanks

**Reply by granwyntan** [reply]
or @Janson-Soh could you upload the entire Zipped Project?

**Comment by granwyntan** [comment]
I suspect this could be an issue with your connections

**Reply by granwyntan** [reply]
You could click on your view controllers in Main.storyboard and then click the Connections Inspector (circle and ball)
<img width="264" alt="Screenshot 2021-07-31 at 12 40 09" src="https://user-images.githubusercontent.com/61010627/127728790-7fdd240e-e68b-4937-8e81-b17f26dd3934.png">
Normally it should look like above, but if you see connections with ⚠️ symbol, please remove them.

**Comment by granwyntan** [comment, marked as answer]
Here are some photos which might be of help
![image](https://user-images.githubusercontent.com/61010627/127728970-f398ca56-d81f-47ea-b78e-f72f92e4f528.png)
![image](https://user-images.githubusercontent.com/61010627/127728978-de1e3aa0-38ac-4df5-944a-0707b9b97a65.png)
![image](https://user-images.githubusercontent.com/61010627/127728979-0c39bdad-18f2-478b-9bb3-13d60d8fb1a1.png)
![image](https://user-images.githubusercontent.com/61010627/127728982-cca3a779-121f-4d0a-b97f-2a915346b6b0.png)

**Reply by granwyntan** [reply]
Kindly done by @jiachenyee

**Reply by JansonJS** [reply]
Ok thanks
